artifactory/utils/commandsummary/buildinfosummary.go

```diff
@@ -2,13 +2,15 @@ package commandsummary
 
 import (
 	"fmt"
+	"net/url"
+	"path"
+	"strings"
+
+	"github.com/jfrog/build-info-go/entities"
```

Comment:
package "github.com/jfrog/build-info-go/entities" is being imported more than once